blockchain/contracts/HelloWorld.sol

```diff
@@ -25,15 +27,18 @@ contract HelloWorld {
         NFTOwnership memory nftOwnership;
         nftOwnership.nftId = nftHash;
         nftOwnership.owner = nftOwner;
+        nftOwnership.droptime = 1637085033;
         nftOwnerships[nftHash] = nftOwnership;
         nfts.push(nftHash);
         return nftHash;
     }
 
-    function buyNFT(uint256 price, uint256 nftHash) public payable {
+    function buyNFT(uint256 price, uint256 nftHash) public payable returns(address) {
+        require(nftOwnerships[nftHash].droptime<=block.timestamp,string(abi.encode("Drop has not started yet! ",(nftOwnerships[nftHash].droptime-block.timestamp),"Seconds left!")));
```

Comment:
I think we should change that to not display seconds but hours:minutes:seconds

Reply:
Due to the fact solidity does not have built-in int->String parsing, nor string concatenation(https://ethereum.stackexchange.com/questions/10811/solidity-concatenate-uint-into-a-string), I would rather not implement any display of remaining time yet, as my approach didnt work anyways. Also the result would only be displayed in the blockchain-viewer like etherscan as far as I know & it should rather be implemented in the frontend in the future.

Reply:
Fine with me :)